[2025-03-10T08:08:15.921+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-10T08:08:17.064+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: validate_all_tables.orders manual__2025-03-10T08:07:23.921116+00:00 [queued]>
[2025-03-10T08:08:17.179+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: validate_all_tables.orders manual__2025-03-10T08:07:23.921116+00:00 [queued]>
[2025-03-10T08:08:17.203+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-10T08:08:17.554+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): orders> on 2025-03-10 08:07:23.921116+00:00
[2025-03-10T08:08:17.618+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'validate_all_tables', 'orders', 'manual__2025-03-10T08:07:23.921116+00:00', '--job-id', '132', '--raw', '--subdir', 'DAGS_FOLDER/validate_all_tables_dag.py', '--cfg-path', '/tmp/tmpl983pfa5']
[2025-03-10T08:08:17.630+0000] {standard_task_runner.py:105} INFO - Job 132: Subtask orders
[2025-03-10T08:08:17.630+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4141) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-10T08:08:17.647+0000] {standard_task_runner.py:72} INFO - Started process 4145 to run task
[2025-03-10T08:08:18.140+0000] {task_command.py:467} INFO - Running <TaskInstance: validate_all_tables.orders manual__2025-03-10T08:07:23.921116+00:00 [running]> on host efacfdd0be48
[2025-03-10T08:08:18.911+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='validate_all_tables' AIRFLOW_CTX_TASK_ID='orders' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T08:07:23.921116+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-10T08:07:23.921116+00:00'
[2025-03-10T08:08:18.923+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-10T08:08:18.926+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-10T08:08:18.928+0000] {logging_mixin.py:190} INFO - Current task name:orders state:running start_date:2025-03-10 08:08:17.093200+00:00
[2025-03-10T08:08:18.929+0000] {logging_mixin.py:190} INFO - Dag name:validate_all_tables and current dag run status:running
[2025-03-10T08:08:18.931+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-10T08:08:19.411+0000] {logging_mixin.py:190} INFO - Orders validation and cleaning successful
[2025-03-10T08:08:19.414+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-10T08:08:19.481+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-10T08:08:19.484+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=validate_all_tables, task_id=orders, run_id=manual__2025-03-10T08:07:23.921116+00:00, execution_date=20250310T080723, start_date=20250310T080817, end_date=20250310T080819
[2025-03-10T08:08:19.820+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-10T08:08:19.823+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-10T08:08:19.825+0000] {logging_mixin.py:190} INFO - Dag name:validate_all_tables queued_at:2025-03-10 08:07:23.959733+00:00
[2025-03-10T08:08:19.826+0000] {logging_mixin.py:190} INFO - Task hostname:efacfdd0be48 operator:PythonOperator
[2025-03-10T08:08:19.912+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-10T08:08:20.086+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-10T08:08:20.110+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
